src/Aspire.Hosting.Azure.AppContainers/ContainerAppExtensions.cs

```diff
@@ -0,0 +1,103 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Azure.Provisioning.AppContainers;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning;
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Hosting.Azure;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for customizing Azure Container App resource.
+/// </summary>
+public static class ContainerAppExtensions
```

Comment:
1. Why is this in a new class?
2. Why does this new class not follow the naming conventions in the rest of the library?

I think this method should go in the existing `AzureContainerAppExtensions` class.

cc @mitchdenny @davidfowl

Reply:
This was intentionally named I think because @mitchdenny was thinking it would eventually be a cdk method. But not sure